[BUG]: docker release build failing in 23.01

### Version

23.01

### Which installation method(s) does this occur on?

_No response_

### Describe the bug.

cmake in meta.yaml needs an update to 3.24 to align with MRC cmake version

### Minimum reproducible example

```shell
./docker/build_container_release.sh
```


### Relevant log output

```shell
#15 774.6 CMake Error at /opt/conda/conda-bld/morpheus-split-23.01.00a/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/cmake/mrc/mrc-config.cmake:60 (cmake_minimum_required):
#15 774.6   CMake 3.24 or higher is required.  You are running version 3.22.3
#15 774.6 Call Stack (most recent call first):
#15 774.6   .cache/cpm/cpm/CPM_7644c3a40fc7889f8dee53ce21e85dc390b883dc.cmake:204 (find_package)
#15 774.6   .cache/cpm/cpm/CPM_7644c3a40fc7889f8dee53ce21e85dc390b883dc.cmake:255 (cpm_find_package)
#15 774.6   build-conda/_deps/rapids-cmake-src/rapids-cmake/cpm/find.cmake:152 (CPMFindPackage)
#15 774.6   cmake/deps/Configure_mrc.cmake:21 (rapids_cpm_find)
#15 774.6   cmake/deps/Configure_mrc.cmake:49 (find_and_configure_MRC)
#15 774.6   cmake/dependencies.cmake:107 (include)
#15 774.6   CMakeLists.txt:72 (include)
#15 774.6
#15 774.6
#15 774.6 -- Configuring incomplete, errors occurred!
#15 774.6 See also "/opt/conda/conda-bld/morpheus-split-23.01.00a/work/build-conda/CMakeFiles/CMakeOutput.log".
#15 774.6 See also "/opt/conda/conda-bld/morpheus-split-23.01.00a/work/build-conda/CMakeFiles/CMakeError.log".
#15 774.8 Traceback (most recent call last):
#15 774.8   File "/opt/conda/bin/conda-mambabuild", line 10, in <module>
#15 774.8     sys.exit(main())
#15 774.8   File "/opt/conda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 256, in main
#15 774.8     call_conda_build(action, config)
#15 774.8   File "/opt/conda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 228, in call_conda_build
#15 774.8     result = api.build(
#15 774.8   File "/opt/conda/lib/python3.8/site-packages/conda_build/api.py", line 180, in build
#15 774.8     return build_tree(
#15 774.8   File "/opt/conda/lib/python3.8/site-packages/conda_build/build.py", line 3093, in build_tree
#15 774.8     packages_from_this = build(metadata, stats,
#15 774.8   File "/opt/conda/lib/python3.8/site-packages/conda_build/build.py", line 2368, in build
#15 774.8     newly_built_packages = bundlers[pkg_type](output_d, m, env, stats)
#15 774.8   File "/opt/conda/lib/python3.8/site-packages/conda_build/build.py", line 1609, in bundle_conda
#15 774.8     utils.check_call_env(interpreter_and_args + [dest_file],
#15 774.8   File "/opt/conda/lib/python3.8/site-packages/conda_build/utils.py", line 402, in check_call_env
#15 774.8     return _func_defaulting_env_to_os_environ('call', *popenargs, **kwargs)
#15 774.8   File "/opt/conda/lib/python3.8/site-packages/conda_build/utils.py", line 382, in _func_defaulting_env_to_os_environ
#15 774.8     raise subprocess.CalledProcessError(proc.returncode, _args)
#15 774.8 subprocess.CalledProcessError: Command '['/usr/bin/bash', '-e', '/opt/conda/conda-bld/morpheus-split-23.01.00a/work/morpheus_build.sh']' returned non-zero exit status 1.
#15 ERROR: executor failed running [/bin/bash -c source activate base &&    git config --global --add safe.directory "*" &&    MORPHEUS_BUILD_PYTHON_STUBS=OFF CONDA_BLD_PATH=/opt/conda/conda-bld CONDA_ARGS="--no-test" ./ci/conda/recipes/run_conda_build.sh morpheus]: exit code: 1
------
 > [conda_bld_morpheus 2/3] RUN --mount=type=cache,id=workspace_cache,target=/workspace/.cache,sharing=locked     --mount=type=cache,id=conda_pkgs,target=/opt/conda/pkgs,sharing=locked     source activate base &&    git config --global --add safe.directory "*" &&    MORPHEUS_BUILD_PYTHON_STUBS=OFF CONDA_BLD_PATH=/opt/conda/conda-bld CONDA_ARGS="--no-test" ./ci/conda/recipes/run_conda_build.sh morpheus:
#15 774.8     packages_from_this = build(metadata, stats,
#15 774.8   File "/opt/conda/lib/python3.8/site-packages/conda_build/build.py", line 2368, in build
#15 774.8     newly_built_packages = bundlers[pkg_type](output_d, m, env, stats)
#15 774.8   File "/opt/conda/lib/python3.8/site-packages/conda_build/build.py", line 1609, in bundle_conda
#15 774.8     utils.check_call_env(interpreter_and_args + [dest_file],
#15 774.8   File "/opt/conda/lib/python3.8/site-packages/conda_build/utils.py", line 402, in check_call_env
#15 774.8     return _func_defaulting_env_to_os_environ('call', *popenargs, **kwargs)
#15 774.8   File "/opt/conda/lib/python3.8/site-packages/conda_build/utils.py", line 382, in _func_defaulting_env_to_os_environ
#15 774.8     raise subprocess.CalledProcessError(proc.returncode, _args)
#15 774.8 subprocess.CalledProcessError: Command '['/usr/bin/bash', '-e', '/opt/conda/conda-bld/morpheus-split-23.01.00a/work/morpheus_build.sh']' returned non-zero exit status 1.
------
ERROR: failed to solve: executor failed running [/bin/bash -c source activate base &&    git config --global --add safe.directory "*" &&    MORPHEUS_BUILD_PYTHON_STUBS=OFF CONDA_BLD_PATH=/opt/conda/conda-bld CONDA_ARGS="--no-test" ./ci/conda/recipes/run_conda_build.sh morpheus]: exit code: 1```
```


### Full env printout

_No response_

### Other/Misc.

_No response_

### Code of Conduct

- [X] I agree to follow Morpheus' Code of Conduct
- [X] I have searched the [open bugs](https://github.com/nv-morpheus/Morpheus/issues?q=is%3Aopen+is%3Aissue+label%3Abug) and have found no duplicates for this bug report

--- a/ci/conda/recipes/morpheus/meta.yaml
+++ b/ci/conda/recipes/morpheus/meta.yaml
@@ -51,7 +51,7 @@ outputs:
         - {{ compiler("c") }}
         - {{ compiler("cxx") }}
         - {{ compiler("cuda") }}
-        - cmake 3.22
+        - cmake 3.24
         - ccache
         - ninja
       host:

